### PR TITLE
#128 - Windows setup file needed for oxd service

### PR DESCRIPTION
**Pull request by duttarnab**
#128 - Windows setup file needed for oxd service
https://github.com/GluuFederation/oxd/issues/128

### CODE REVIEW

File: oxd-server/doc/license.txt [hunk 1]
```diff
@@ -0,0 +1,651 @@
+GNU Affero General Public License
```
Comment (yuriyz) [inline oxd-server/doc/license.txt]: @duttarnab Do we really need this license file here?

File: oxd-server/pom.xml [hunk 2]
```diff
@@ -166,7 +167,25 @@
                     <failOnNoGitDirectory>false</failOnNoGitDirectory>
                 </configuration>
             </plugin>
-
+            <plugin>
```
Comment (yuriyz) [inline oxd-server/pom.xml]: Lets add it into windows-build maven profile, so it is not activated by default.

File: oxd-server/doc/license.txt (hunk 1 above)
Comment (duttarnab) [reply oxd-server/doc/license.txt]: Removed this file

File: oxd-server/pom.xml (hunk 2 above)
Comment (duttarnab) [reply oxd-server/pom.xml]: Created `windows-build` maven profile

Comment (yuriyz) [reply oxd-server/pom.xml]: @duttarnab why is it blank? all windows-build related configuration should go inside it. So if on maven package we invoke download and build it must be done within `windows-build` profile. You may wish to read this first http://maven.apache.org/guides/introduction/introduction-to-profiles.html

Comment (duttarnab) [reply oxd-server/pom.xml]: We have added windows-build related configuration in `windows-build` profile.

**Pull request merged**